java-runtime-builder-docker/src/main/resources/docker/Dockerfile

```diff
@@ -1,29 +1,21 @@
-FROM gcr.io/google-appengine/openjdk
-
-# install utilities
-RUN apt-get -q update \
-  && apt-get -q -y --no-install-recommends install curl \
-  && apt-get clean \
-  && rm /var/lib/apt/lists/*_*
-
-# install maven
-ARG MAVEN_VERSION=3.3.9
-RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
-  && curl -fsSL https://apache.osuosl.org/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz \
-    | tar -xzC /usr/share/maven --strip-components=1 \
-  && ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
-ENV M2_HOME /usr/share/maven
-
-# install gradle
+# Use wget container to download gradle
+FROM gcr.io/cloud-builders/wget as wget
 ARG GRADLE_VERSION=3.4
 RUN mkdir -p /usr/share/gradle \
-  && curl -fsSLO https://services.gradle.org/distributions/gradle-$GRADLE_VERSION-bin.zip \
-  && unzip -qq gradle-$GRADLE_VERSION-bin.zip \
-  && rm gradle-$GRADLE_VERSION-bin.zip \
+  && wget -nv -O gradle-$GRADLE_VERSION-bin.zip https://services.gradle.org/distributions/gradle-$GRADLE_VERSION-bin.zip
```

Comment:
Why not just use `ADD https://services.gradle.org/distributions/gradle-$GRADLE_VERSION-bin.zip gradle-$GRADLE_VERSION-bin.zip`? The `FROM` line can then just be `scratch`?